### ISSUE TITLE
[FEATURE] Delete event

### ISSUE THREAD

**Issue by justynpride** [EstebanJoseSantos/myAttendize#1]
I thought it worth continuing our conversation here - my Repo is https://github.com/justynpride/Attendize, and the delete feature is under jp-delete branch. Don't know if it is of help.

**Comment by EstebanJoseSantos** [comment]
Yep, I just finished updating the my repositories to the latest official version.  So any isse we can discuss here

**Comment by justynpride** [comment]
Great. I see you pushed to the main repo. I hope it gets added in as it really is a helpful feature. 

I'm always slightly surprised with Attendize with the lack of delete features. You can't delete users, tickets or events. I appreciate you don't want to delete things that are actively used, but delete gives good house keeping. 

Would love to hear about your use scenarios for Attendize.

**Comment by justynpride** [comment]
Interestingly enough there is some code of deleting to tickets but nothing implemented on the edit ticket modal which would be the natural place to have a delete ticket option.  - app/Http/Controllers/EventTicketsController.php lines 168-208.

**Comment by justynpride** [comment]
I've made some changes and my main branch is **delete**. In it is your pull request, but also my adaption for tickets. It does delete the ticket, but could probably be done better. I wonder whether a hard delete of the entry rather than soft delete is better.
